ssdc-rm-common-entity-model/src/main/java/uk/gov/ons/ssdc/common/model/entity/ActionRule.java

```diff
@@ -60,6 +60,8 @@ public class ActionRule {
   @Column(columnDefinition = "jsonb")
   private Object uacMetadata;
 
+  @Column private int selectedCaseCount;
```

Comment:
I think the column should be empty if it hasn't been run yet. There's the potential for aciton rules to select no cases so I think it'd be better for the column to be empty before it runs. Then we can see how many cases it selects